cmd/operator/run.go

```diff
@@ -67,6 +67,18 @@ func runProtoform(configPath string) {
 		panic(err)
 	}
 
+	// Log Kubernetes version
```

Comment:
@jdartigalongue 
whether it should be either or?

Reply:
I personally think that it's relevant to get both the kubernetes and openshift version even for Openshift users. The oc utility also prints both versions.